lib/libc/minimal/include/fcntl.h

```diff
@@ -8,13 +8,14 @@
 #define ZEPHYR_LIB_LIBC_MINIMAL_INCLUDE_FCNTL_H_
 
 #define O_CREAT    0x0200
+#define O_APPEND   0x0400
 #define O_EXCL     0x0800
 #define O_NONBLOCK 0x4000
 
 #define F_DUPFD 0
 #define F_GETFL 3
 #define F_SETFL 4
 
-int open(const char *name, int flags);
+int open(const char *name, int flags, ...);
```

Comment:
Please add a note to the commit message that you change open() signature. Note that while this change is correct (and I'm ok with it being piggy-backed here), nothing in the rest of your patch relies or uses this change. So, if you follow a "don't make unrelated/unnecessary changes" strictly, you could as well drop this change (and leave it for later, for dedicated commit, and detailed commit message).

Reply:
OK, will move it to separate commit with proper comment. The change has been done, because some of headers come from SDK and there the `open` is declared with ellipsis, which caused conflict with Zephyr definition of function.

Reply:
> OK, will move it to separate commit with proper comment. 

That's how I'd do it, so appreciated. Otherwise, I agree it's the right change to do.

Reply:
Done.